proxies/live/apiproxy/policies/RaiseFault.401OperationOutcomeInsufficientIal.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<RaiseFault async="false" continueOnError="false" enabled="true" name="RaiseFault.401OperationOutcomeInsufficientIal">
+    <DisplayName>RaiseFault.401OperationOutcomeInsufficientIal</DisplayName>
+    <Properties/>
+    <FaultResponse>
+        <Set>
+            <Headers/>
+            <Payload contentType="application/fhir+json" variablePrefix="%" variableSuffix="#">{ "resourceType": "OperationOutcome", "meta": { "lastUpdated": "%current_timestamp#", "profile" : [ "%op_outcome_fhir_profile#" ] }, "issue": [ { "severity": "error", "code": "forbidden", "details": { "coding": [ { "system": "https://fhir.nhs.uk/CodeSystem/NHSD-API-ErrorOrWarningCode", "code": "ACCESS_DENIED" } ] }, "diagnostics": "We couldn't verify your identity. Contact your local Registration Authority or IT department for help." } ] }</Payload>
```

Comment:
Outside of the diagnostic message this looks very similar to our existing error payload within `AssignMessage.OAuthPolicyOperationOutcomeErrorResponse`, could the existing `AssignMessage.OAuthPolicyOperationOutcomeErrorResponse` be generalised so that the same response can be used in both places?

Reply:
Done.